spec/xpm_ruby/client/contact/update_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+
+module XpmRuby
+  module Client
+    RSpec.describe(Contact) do
```

Comment:
Do we need to put this in its own spec file?

Reply:
Probably not at this point just a habit TBH.

I'm following up with a few cleanup PR's will keep that in mind.

Reply:
Ah ok. My preference is not to break things up until there is a need to do so (because then you get the opposite problem which is file system explosion). But maybe that's just me...
;)

Tony